samples/snippets/core/system-text-json/csharp/Utf8ReaderPartialRead.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+
+namespace SystemTextJsonSamples
+{
+    public class Utf8ReaderPartialRead
+    {
+        public static void Run()
+        {
+            var jsonString = @"{
+                ""Date"": ""2019-08-01T00:00:00-07:00"",
+                ""Temperature"": 25,
+                ""TemperatureRanges"": {
+                    ""Cold"": { ""High"": 20, ""Low"": -10 },
+                    ""Hot"": { ""High"": 60, ""Low"": 20 }
+                },
+                ""Summary"": ""Hot"",
+            }";
+
+            byte[] bytes = Encoding.UTF8.GetBytes(jsonString);
+            var stream = new MemoryStream(bytes);
+
+            var buffer = new byte[10];
+            var span = new Span<byte>(buffer);
```

Comment:
You don't need this local. You can pass `buffer` directly to all the places that take `span`.